src/Microsoft.Framework.CommandLineUtils/CommandLine/CommandLineApplication.cs

```diff
@@ -322,6 +322,7 @@ public void ShowHelp(string commandName = null)
                 headerBuilder.Insert(6, string.Format(" {0}", cmd.Name));
             }
 
+            var version = string.Format("Microsoft .NET Execution environment v{0}{1}{1}", VersionGetter(), Environment.NewLine);
```

Comment:
`CommandLineApplication` is shared by a lot of command-line tools. We shouldn't show this string when you check help info of `kpm` and `dnx`.

I suggest we add a `FullName` property, which can be set by users of `CommandLineApplication`